.github/workflows/build-and-push-image.yml

```diff
@@ -173,9 +173,25 @@ jobs:
         with:
           name: screenshots-${{ needs.set-env.outputs.environment }}
           path: Dfe.PrepareConversions/Dfe.PrepareConversions.CypressTests/cypress/screenshots
-          
+
+      - name: Generate report
+        if: always()
+        run: |
+            mkdir mochareports
+            npm run generate:html:report    
+
+            - name: Upload report
```

Comment:
Check the formatting of this, I suspect it'll cause the pipeline to fail due to incorrect indentation

Reply:
I checked it, it seems to be okay. but I can reformat on the next push if there is any problems.